tidy3d/components/scene.py

```diff
@@ -993,10 +1084,22 @@ def plot_structures_property(
                         shape=shape,
                         ax=ax,
                         property="doping",
+                        # Note: I omitted `transpose` because we took care of that earlier.
                     )
                 else:
                     self._pcolormesh_shape_doping_box(
-                        x, y, z, alpha, medium, property_min, property_max, shape, ax, property
+                        x,
+                        y,
+                        z,
+                        alpha,
+                        medium,
+                        property_min,
+                        property_max,
+                        shape,
+                        ax,
+                        property,
+                        transpose=transpose,
+                        transpose_shape=False,  # We took care of transposing the shape earlier.
```

Comment:
I can confirm that the function behaves correctly when it reaches this line and invokes `_pcolormesh_shape_doping_box()`'.